src/AutoBuild.jl

```diff
@@ -1121,6 +1121,9 @@ function build_jll_package(src_name::String,
             end
 
             print(io, """
+            # Inform that the wrapper is available for this platform
+            is_available() = tue
```

Comment:
```suggestion
            is_available() = true
```

Reply:
Isn't the usual Julia convention to not have an underscore after `is`, i.e., `isavailable`? Mind you, I am not complaining; personally, I prefer `is_...` but it doesn't seem to be used in Julia otherwise?

Reply:
That's a convention in the Julia's code base, here we have a lot of functions with underscores.

Any other suggestion about the name? Is `is_available` clear enough? Alternatives?